test/components/info2/info2.scss

```diff
@@ -0,0 +1,5 @@
+.header {
+    .header--main-info-2 {
```

Comment:
Did you intend to generate  `.header .header--main-info-2` ?
Seems like the top level can go away.